drupal/sites/all/features/user_profiles/user_profiles.features.content.inc

```diff
@@ -492,9 +492,9 @@ Zimbabwe|Zimbabwe',
       'default_value_php' => NULL,
       'label' => 'Your opinions about this project',
       'weight' => '4',
-      'description' => 'Tell us your thoughts about Einstein@Home
+      'description' => 'Tell us your thoughts about ' . variable_get('site_name', '') . '
```

Comment:
Somehow I missed that this string should be translated. As a result I will close this PR. And then add `bts()` to these strings. Then I will add those changes as part of #1834. Thanks for finding this.